portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase

```diff
@@ -268,6 +268,40 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-146160 : Verify Admin can enable local staging on the inherited site even site template propagation is disabled.The testcase is blocked by"
```

Comment:
Missing blocked ticket reference from the description

Reply:
thanks and done